prow/apis/prowjobs/v1/types.go

```diff
@@ -1046,6 +1046,14 @@ func (r Refs) String() string {
 	return strings.Join(rs, ",")
 }
 
+func (r Refs) OrgRepoString() string {
+	orgRepo := r.Org
```

Comment:
Nit: It would be cleaner to do

```
if len(r.Repo) != 0 { return r.Org + "/" + r.Repo }
return r.Org
```

and avoid the intermediate `orgRepo` variable.

Reply:
Agreed, fixed!